### ISSUE TITLE
Loading Bug

### ISSUE THREAD

**Issue by TheCherry** [JACoders/OpenJK#780]
Hi,

I have a loading bug in OpenJK.

Tested Versions:
openjk-2015-11-30-c8c64c4b-linux
openjk-2015-11-30-c8c64c4b-linux-64

System:
Linux Mint 17.1 - 64bit

Problem:
If I loading a game (from ingame) some objects don't interact correct. As example the tree at start cant be break. Here a video: https://youtu.be/DFTr8XYZf88

(The crash at the end of the video is course I try to load a save file from 64bit version in the 32bit version.)

**Comment by ensiform** [comment]
Saves are not cross arch compatible.

**Comment by TheCherry** [comment]
Yes thought that. Just wanted to explain, why its crash at the end of the video.

**Comment by yomli** [comment]
Same here. Seems related to issue #257. 
Version 2016-01-04-db94c70b-linux-64 on Linux Mint 17.3.

My workaround : restart and load.

**Comment by xycaleth** [comment]
I believe this is a duplicate.of #257.and #295. I fixed it in e8fc63788, but it was potentially broken again in bace05c9b3d when the `-fvisibility=hidden -fvisibility-inlines-hidden` were removed for GCC builds...
